test/nut/commands/force/org/status.nut.ts

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as fs from 'fs';
```

Comment:
I think this file should become `.sandboxNut.ts` so that it's only ran when we're testing SBs when `[sb-nuts]` is present in a commit, or PR